src/framing/mod.rs

```diff
@@ -66,7 +66,8 @@ impl MLSPlaintext {
             content,
             signature: Signature::new_empty(),
         };
-        mls_plaintext.sign(ciphersuite, signature_key, context);
+        let serialized_context = context.encode_detached().unwrap();
```

Comment:
Shouldn't the error get propagated here instead of panicking? The `context` is input and might not be well-formed.

Reply:
The codec code throws an error when slices are too long to be encoded with a certain length header (e.g. a slice of 257 elements encoded with a `u8` header).
If we ant to make sure the group context doesn't have this problem, we should check it when constructing it, not when serializing it.
Sanitizing input is something we should improve anyway, so that we don't have to worry about it internally. I filed #115 for that.